File: day1/javascript/03-arrays.js
let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
console.log("all array's elements", arr);

console.log("element by element");
for (let i = 0; i < arr.length; i++) {
  console.log(arr[i]);
}

arr[0] = 99;
console.log(arr);

console.log("using array's forEach method");
arr.forEach((element) => console.log(element));

let doubleVals = arr.map((element) => {
  return element * 2;
});
console.log(doubleVals);

console.log(
  arr.findIndex((element) => {
    return element === 99;
  })
);

File: day6/03-arrays.go
package main

import "fmt"

func main() {
	var arr = []int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}
	fmt.Println("all array's elements", arr)

	fmt.Println("element by element")
	// using counter loop
	for i := 0; i < len(arr); i++ {
		fmt.Println(arr[i])
	}

	// using for range
	for index, value := range arr {
		fmt.Println(index, value)
	}

	arr[0] = 99
	fmt.Println(arr)

}
